Chapter_9/ex9.rb

```diff
@@ -0,0 +1,13 @@
+# Here's some new strange stuff, remember type it exactly.
+
+days = 'Mon Tue Wed Thu Fri Sat Sun'
+months = "Jan\nFeb\nMar\nApr\nMay\nJun\nJul\nAug"
+
+puts "Here are the days: #{days}"
+puts "Here are the months: #{months}"
+
+puts
```

Comment:
Try re-writing the code exactly as he has it in the book. Then where he has triple quotes, use single quotes instead. Does it still work? Try it out